eng/helix/configure-helix-machine.ps1

```diff
@@ -0,0 +1,18 @@
+# This file prepares the helix machine for our tests runs
+
+$dotnetLocation = Join-Path (Split-Path -Parent $script:MyInvocation.MyCommand.Path) "dotnet"
+
+Write-Output "Adding dotnet location to path: $dotnetLocation"
+
+# Prepend the path to ensure that dotnet.exe is called from there.
+$env:path="$dotnetLocation;$env:path"
+
+# Don't look outside the TestHost to resolve best match SDKs.
```

Comment:
Why do we still use the concept of a TestHost -  shouldn't this instead be relying upon Arcade's multi-framework support and running off of `Microsoft.NetCore.App` version corresponding to `RuntimeFrameworkVersion` + project references to WPF assemblies, and impliict support provided by Arcade for `DOTNET_MULTILEVEL_LOOKUP=0` etc. when `ContinousIntegrationBuild==true`?

Reply:
i'm taking a walk, crawl, run approach here. We already know how to run DRTs as they are, and we're now trying to move that into Helix. I'd rather not try to develop the new method of how to build and run the DRTs at the same time that we're also doing something else. that is just going to slow down the entire process.

Reply:
We should use the right approach in production codebases. 

The right approach is is in fact the easier one, and I don't believe it fits the mode of walk-crawl-run. In my estimation, the changes required to build these projects using `Microsoft.Net.Sdk` should take a couple of hours of work at most (for System.Xaml DRTs') - I know because I tried doing it once as a trial until I had to stop due to the lack of `multi-framework` and `InternalMarkupCompilation` support. They are both available to you readily now.